src/views/StandaloneDirectorsFiling.vue

```diff
@@ -402,10 +404,10 @@ export default {
 
     // NB: filing id of 0 means "new"
     // otherwise it's a draft filing id
-    this.filingId = +this.$route.params.id // number
+    this.filingId = +this.$route.params.id // number (may be NaN)
```

Comment:
Just curious how this differs from the previous filing where it sets the id to 0?

Reply:
0 means new filing

> 0 means resume draft

Unspecified (ie, NaN) means you got to this page without setting the route param (eg, by returning to a bookmark) so you get rerouted to the dashboard.

Reply:
ahhh right right, i saw that before. Right on, thanks for clarifying!